app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -104,6 +107,7 @@ class ProfileEditActivityTest {
     }
   }
 
+  // Failing in other Devices too. ActionBar does not have the text "Ben". It has "Oppia"
   @Test
   fun testProfileEditActivity_configurationChange_startActivityWithUserProfile_checkUserInfoIsDisplayed() {
```

Comment:
This test is failing in other devices too. ActionBar does not have the text "Ben". It has "Oppia".